src/bin/bano2mimir.rs

```diff
@@ -92,6 +93,13 @@ async fn run(
         .context(ElasticsearchConnectionSnafu)
         .map_err(Box::new)?;
 
+    tracing::info!("Connected to elasticsearch.");
+
+    // Update all the template coponents and indexes
```

Comment:
A small typo that is repeated in most files :stuck_out_tongue: 

```suggestion
    // Update all the template components and indexes
```

Reply:
:+1: